soql-server-pg/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# Set everything to be logged to the file target/unit-tests.log
```

Comment:
Hrm.  We may need to figure out some different way to deal with different development tools here.  For example, when I run tests in IntelliJ then I really want messages to go to stdout because IntelliJ automatically groups them under the right tests so I can easily see the logs associated with a failing test.